setup/resources.py

```diff
@@ -143,9 +143,14 @@ def run(self, opts):
             if err.errno != errno.ENOENT:
                 raise
             raw = b''
-        nraw = download_securely('https://curl.haxx.se/ca/cacert.pem')
-        if not nraw:
-            raise RuntimeError('Failed to download CA cert bundle')
+        env = os.getenv('CACERT')
+        if env:
+            with open(env, 'rb') as f:
+                nraw = f.read()
+        else:
+            nraw = download_securely('https://curl.haxx.se/ca/cacert.pem')
+            if not nraw:
+                raise RuntimeError('Failed to download CA cert bundle')
```

Comment:
Just do not run this command at all, if you've copied it in place yourself...

Reply:
I just keep all original logics in else block. I planned to do cleanup work in following commits if this PR has been accpeted.